exports.js

```diff
@@ -989,7 +989,9 @@ module.exports = {
         'eventHubMinimumTLSversion'     : require(__dirname + '/plugins/azure/eventhub/eventHubMinimumTLSversion.js'),
 
         'accessLogsEnabled'             : require(__dirname + '/plugins/azure/frontdoor/accessLogsEnabled.js'),
-        'frontDoorMinimumTlsVersion'    : require(__dirname + '/plugins/azure/frontdoor/frontDoorMinimumTlsVersion.js')
+        'frontDoorMinimumTlsVersion'    : require(__dirname + '/plugins/azure/frontdoor/frontDoorMinimumTlsVersion.js'),
+
+        'amsStorageAccountIdentityEnabled': require(__dirname + '/plugins/azure/mediaServices/amsStorageAccountIdentityEnabled.js')
```

Comment:
```suggestion
        'amsStorageAccountIdentity': require(__dirname + '/plugins/azure/mediaServices/amsStorageAccountIdentityEnabled.js')
```
Filename could just be amsStorageAccountIdentity